### DISCUSSION TITLE
@NameIsA

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#640] [(A) Anything]
@NameIsA uh i tried the live chat and this is what it said request to be verified on the fandom prodigy wiki live chat

**Comment by ghost** [comment]
bruh the live chat is dead

**Reply by ghost** [reply]
it would be alive but fandom banned me for monetizing it

**Comment by irhrhd** [comment]
Well then fuk fandom they can suck it
